guardian/media-atom-maker
# Bump packages to fix vulnerabilities

## What does this change?

This bumps packages where the version bump fixes high priority vulnerabilities.

This PR brings the number of vulnerabilities in our dependencies from 19 to 7.

Follow-up PRs will make less trivial fixes to attempt to solve the remaining high priority vulnerabilities.

### Before:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/34686302/217229025-67a7e186-8826-4f03-88bd-498b6aa22f62.png">

### After:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/34686302/217229111-3ad98a9e-dbf7-480f-a3d3-fc04f4ef8554.png">


## How to test

Run locally according to the instructions in the readme or deploy to CODE. Does the application build and run as expected?
